codec_specs.md

```diff
@@ -88,6 +88,10 @@ The values of `BlockAddID` that are 2 of greater have no semantic meaning, but s
 associate the `BlockMore Element` with a `BlockAdditionMapping` of the associated Track.
 See (#block-additional-mapping) on Block Additional Mappings for more information.
 
+The value of 4 for `BlockAddID` SHOULD NOT be used when `BlockAddIDType` is not 4,
```

Comment:
>The value of 4 for `BlockAddID`

I think this should be `BlockAddIDValue` instead, which implies all the relevant `BlockAddID` in the track need to follow suit.

Reply:
> I think this should be BlockAddIDValue instead, which implies all the relevant BlockAddID in the track need to follow suit.

The section is about `BlockAddID` so I prioritize it, but I added an explicit outcome on `BlockAddIDValue`. Reopen if not fine.